Empty user fitting directories with specifically lengthed fitting names throwing errors

Using a list of user defined fitting directories, with one directory that is empty, and a custom fitting with a name exactly 20 characters in length (and potentially other lengths), the control logic determining whether to throw an error or to do nothing and continue to check the next user defined fitting directory is no longer valid.

The problem is here: https://github.com/apigee-127/bagpipes/blob/c14ffd2605b71d3af8931388010ec5d80706d117/lib/fittingTypes/user.js#L26

Using a user defined fitting directory that doesn't exist, the catch block is executed, and `err.message` is similar to `Cannot find module '/Users/johnsmith/dev/bagpipes/src/fittings/this_is_20_character'`. The control logic then checks that there is a path separator before the fitting name (`err.message[fittingIndex - 1] === path.sep`). This passes. It then checks if the error message character at the index of the fitting name length, which in this case is 20, is not a path separator (`err.message[fittingDef.name.length] !== path.sep`), which it is, because that character is the beginning of our URL string in the error message. Checking for this arbitrary length in the error message makes no sense.

If I understood what the control logic was attempting to control, I would submit a PR for a fix, but I honestly have no idea what checking the error message index at the length of the fitting name could possibly be doing.

The following test, which demonstrates the above problem, fails whether or not the actual fitting exists in `./fixtures/fittings`.

```js
// test/bagpipes.js

it('should attempt to load a fitting from all user defined fitting directories', function(done) {
  // Using 'src/fittings' throws an error, and ensures catch logic is ran
  var userFittingsDirs = [ 'src/fittings', path.resolve(__dirname, './fixtures/fittings') ];
  var pipe = [ 'this_is_20_character' ]; // The name of the fitting is 20 characters, the exact index in which the control logic is invalid
  var bagpipes = Bagpipes.create({ pipe: pipe }, { userFittingsDirs: userFittingsDirs });
  var context = {};
  bagpipes.play(bagpipes.getPipe('pipe'), context);
  context.output.should.eql('test');
  done();
});
```

--- a/lib/fittingTypes/user.js
+++ b/lib/fittingTypes/user.js
@@ -22,8 +22,8 @@ module.exports = function createFitting(pipes, fittingDef) {
     } catch (err) {
       if (err.code !== 'MODULE_NOT_FOUND') { throw err; }
       var pathFromError = err.message.match(/'.*?'/)[0];
-      var fittingIndex = err.message.indexOf(fittingDef.name);
-      if (err.message[fittingIndex - 1] === path.sep && err.message[fittingDef.name.length] !== path.sep) {
+      var split = pathFromError.split(path.sep);
+      if (split[split.length - 1] === fittingDef.name + "'") {
         debug('no user fitting %s in %s', fittingDef.name, dir);
       } else {
         throw err;

